TowerDefense/GameObjects/Enemy.cs

```diff
@@ -54,6 +63,21 @@ public void damage(int damage)
 
         public bool Update(GameTime gameTime)
         {
+            if (HP <= 0)
+            {
+                color.R -= 15;
+                color.G -= 15;
+                color.B -= 15;
+                if (color == Color.Black)
+                {
+                    return true;
```

Comment:
It took me a minute, but I think I get it. This is a fade to black situation. 
```
if (HP <= 0)
    return FadeToBlack();
```

Also, this is a throwback to your previous PR where Update returning true means: Remove the object from the game. I think we set a Destroy bool in our game, now that I think of it. Then we have an update and cleanup cycle in the main Update method.